tests/test_django/urls.py

```
@@ -52,6 +119,83 @@
+    @override
+    def get_filter_for_list(self, request: RequestDetails[TDjangoModel]) -> Q:
+        """Return the queryset filter that applies to the list operation."""
+        user = cast(User, request.request.user)
+
+        if user.username == ADMIN_USER:  # pyright: ignore [reportUnknownMemberType]
+            return Q()
+
+        if user.username == STANDARD_USER:  # pyright: ignore [reportUnknownMemberType]
+            return Q(created_by=user)
+
+        return request.model_class._default_manager.none()  # type: ignore[return-value]
```

Ruff (0.8.2)

Private member accessed: `_default_manager`

(SLF001)